commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java

```diff
@@ -40,21 +44,35 @@
 
   public static final String COMMAND_NAME = "eth2";
 
-  @Spec CommandSpec spec;
+  @Spec CommandSpec commandSpec;
+
+  @CommandLine.Option(
+      names = {"-n", "--network"},
+      paramLabel = "<NETWORK>",
+      description = "Represents which network to use.",
```

Comment:
Mention what values it could, the enum values? And should also mention that this could be a filename

Reply:
This one is copied verbatim from Teku codebase as well. Let me update the description to mention URI or directory path. However, network config names (i.e. mainnet, pyrmont, prater etc.) are subject to change. https://docs.teku.consensys.net/en/latest/Reference/CLI/CLI-Syntax/#network